crates/sui-node/src/admin.rs

```diff
@@ -80,6 +80,7 @@ const NODE_CONFIG: &str = "/node-config";
 const RANDOMNESS_PARTIAL_SIGS_ROUTE: &str = "/randomness-partial-sigs";
 const RANDOMNESS_INJECT_PARTIAL_SIGS_ROUTE: &str = "/randomness-inject-partial-sigs";
 const RANDOMNESS_INJECT_FULL_SIG_ROUTE: &str = "/randomness-inject-full-sig";
+const RECONFIGURE_TRAFFIC_CONTROL: &str = "/reconfigure-traffic-control";
```

Comment:
nit: just `traffic-control` is probably sufficiently clear?

Reply:
fair